packages/asset-service/src/generateAssetData/Utils.test.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  AssetDataSource,
+  BaseAsset,
+  ChainTypes,
+  ContractTypes,
+  NetworkTypes,
+  TokenAsset
+} from '@shapeshiftoss/types'
+
+import { filterBlacklistedAssets } from './generateAssetData'
+
+jest.mock(
+  './blacklist.json',
+  () => [
+    'bip122:000000000019d6689c085ae165831e93/slip44:0',
+    'eip155:1/erc20:0x426ca1ea2406c07d75db9585f22781c096e3d0e0'
+  ],
+  { virtual: true }
+)
+
+const BtcAsset: BaseAsset = {
+  caip19: 'bip122:000000000019d6689c085ae165831e93/slip44:0',
+  caip2: 'bip122:000000000019d6689c085ae165831e93',
+  chain: ChainTypes.Bitcoin,
+  dataSource: AssetDataSource.CoinGecko,
+  network: NetworkTypes.MAINNET,
+  symbol: 'BTC',
+  name: 'Bitcoin',
+  precision: 8,
+  slip44: 0,
+  color: '#FFFFFF',
+  secondaryColor: '#FFFFFF',
+  icon: 'https://assets.coincap.io/assets/icons/256/btc.png',
+  explorer: 'https://live.blockcypher.com',
+  explorerAddressLink: 'https://live.blockcypher.com/btc/address/',
+  explorerTxLink: 'https://live.blockcypher.com/btc/tx/',
+  sendSupport: false,
+  receiveSupport: false
+}
+
+const MNEAsset: TokenAsset = {
```

Comment:
It doesn't matter which token it is, that's an implementation detail, from a high-level perspective the asset is like any ERC20 and that's what's being tested
```suggestion
const ERC20Asset: TokenAsset = {
```